src/app/wallet/wallet.component.scss

```diff
@@ -111,20 +111,14 @@
     letter-spacing: 2px;
     margin-bottom: 34px;
   }
-  &__error {
+  &__grayed-out {
     color: $icons-color;
   }
   .left {
     &__desc {
       line-height: 20px;
       letter-spacing: 0.18px;
     }
-    &__transaction-type {
-      font-weight: bold;
-    }
-    &__conjuctor {
-      display: none;
-    }
```

Comment:
This was used to hide conjuctors on mobile.
If you look at design, these are hidden on mobile version